src/bin/fuzzer.c

```diff
@@ -137,8 +137,24 @@ int LLVMFuzzerInitialize(int *argc, char ***argv)
 		}
 	}
 
-	if (!dict_dir) dict_dir = DICTDIR;
-
+	/*	if (!dict_dir) dict_dir = int; -- removed to match oss-fuzz patch    */
+
+        DICTDIR free_dict = 0;
```

Comment:
Maybe is it `int free_dict = 0;`?

Reply:
That looks like a desirable change.  Thank you.